account/guides/about.md

```diff
@@ -0,0 +1 @@
+Hello world
```

Comment:
Do we want to put something like __Coming Soon!__ here? instead of hello world.  

Or something else?

Reply:
lol :) i updated it. forgot i needed a place holder...